src/e2e-test/java/io/cdap/plugin/dataplex/locators/CdfBigQueryPropertiesLocators.java

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
Use `CdfBigQueryPropertiesLocators.java` from e2e framework. No need to add here.

Reply:
Same comment for `CdfGCSLocators.java`